src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ServiceOfNoticeDetails.cshtml

```diff
@@ -6,8 +6,14 @@
 @model OrganisationDetailsTabsViewModel
 
 @{
-    var @title = "Service of notice details";
-    ViewBag.Title = @title + " - GOV.UK";
+    var title = "Service of notice details";
+    ViewBag.Title = title + " - GOV.UK";
+
+    var hasSubmission = Model.SmallProducerSubmissionData.CurrentSubmission != null;
```

Comment:
@stevelee-civica  is it possible here to check Model.ServiceOfNoticeViewModel != null ?

Reply:
updated